bot/seasons/evergreen/battleship.py

```diff
@@ -324,64 +344,54 @@ def already_playing(self, player: discord.Member) -> bool:
         )
         self.waiting.append(ctx.author)
-        await announcement.add_reaction("\U0001f64b")
-
-        def predicate(reaction: discord.Reaction, user: discord.Member):
-            if self.already_playing(ctx.author):  # If they've joined a game since requesting a player 2
-                raise asyncio.TimeoutError
-            if (
-                user.id not in [ctx.me.id, ctx.author.id]
-                and str(reaction.emoji) == "\U0001f64b"
-                and reaction.message.id == announcement.id
-            ):
-                if self.already_playing(user):
-                    self.bot.loop.create_task(ctx.send(f"{user.mention} You're already playing a game!"))
-                    return False
-                return True
-            return False
+        await announcement.add_reaction(HAND_RAISED_EMOJI)
 
         try:
-            _reaction, user = await self.bot.wait_for("reaction_add", check=predicate, timeout=60.0)
+            _reaction, user = await self.bot.wait_for(
+                "reaction_add",
+                check=partial(self.predicate, ctx, announcement),
+                timeout=60.0
+            )
         except asyncio.TimeoutError:
             self.waiting.remove(ctx.author)
             await announcement.delete()
-            if self.already_playing(ctx.author):
-                return
-            return await ctx.send(f"{ctx.author.mention} Seems like there's noone here to play...")
-        else:
-            await announcement.delete()
-            self.waiting.remove(ctx.author)
-            try:
-                if self.already_playing(ctx.author):
-                    return
-                game = Game(self.bot, ctx.channel, ctx.author, user)
-                self.games.append(game)
-                await game.start_game()
-            except discord.Forbidden:
-                await ctx.send(
-                    f"{ctx.author.mention} {user.mention}"
-                    "Game failed. This is likely due to you not having your DMs open. Check and try again."
-                )
-                self.games.remove(game)
-            except Exception:
-                # Unforseen error so the players aren't stuck in a game
-                await ctx.send(f"{ctx.author.mention} {user.mention} An error occured. Game failed")
-                self.games.remove(game)
-                raise
+            return await ctx.send(f"{ctx.author.mention} Seems like there's no one here to play...")
+
+        await announcement.delete()
+        self.waiting.remove(ctx.author)
+        if self.already_playing(ctx.author):
+            return
+        try:
+            game = Game(self.bot, ctx.channel, ctx.author, user)
+            self.games.append(game)
+            await game.start_game()
+            self.games.remove(game)
+        except discord.Forbidden:
+            await ctx.send(
+                f"{ctx.author.mention} {user.mention}"
```

Comment:
Add a space between the mention and the message.

```suggestion
                f"{ctx.author.mention} {user.mention} "
```